jaxrs-tck/pom.xml

```diff
@@ -29,8 +29,8 @@
     <url>https://github.com/eclipse-ee4j/jaxrs-api</url>
 
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
I would prefer moving to Java 11 explicitly in a second PR, and do that switch for all our code then.

Reply:
The main requirement here is that I am using the Java 11 HTTP Client for testing, so the change would need to be done either in this PR or in a PR that is merged before mine.

Reply:
The EF's rules are clear and publicly available: They demand ONLY the initial year, but do NOT require the second, updated year. This IS the official EF statement. So if nobody wants to have it updated, the EF is fine if we stick with the original single-year variant.